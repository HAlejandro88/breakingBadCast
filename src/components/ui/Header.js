import React from 'react';
import logo from '../../img/logo.png';

const header = () => {
    return (
        <header className="center">
            <img src={ logo } alt='logoP' />
        </header>
    )
}

export default header
