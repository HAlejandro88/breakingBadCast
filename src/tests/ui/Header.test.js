import React from 'react';
import '@testing-library/jest-dom';
import { shallow } from 'enzyme';
import Header from '../../components/ui/Header';
import logo from '../../img/logo.png';


describe('pruebas en ele componente <Header/>', () => {
    const wrapper = shallow(<Header logo={logo} />)
    
    test('pruebas contra el snapShot', () => {
        expect(wrapper).toMatchSnapshot();
    });

    test('pruebas que tiene el componente', () => {
        const wrapper = shallow(<Header logo={logo} />)
        expect(wrapper.find('img').prop('src')).toBe(logo);
    });
});