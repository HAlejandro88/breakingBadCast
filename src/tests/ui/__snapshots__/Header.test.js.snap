// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas en ele componente <Header/> pruebas contra el snapShot 1`] = `
<header
  className="center"
>
  <img
    alt="logoP"
    src="logo.png"
  />
</header>
`;
